exercises/shared/.docs/tests.md

```diff
@@ -0,0 +1,30 @@
+# Running tests
+
+Execute the tests with:
+
+```bash
+$ mix test
+```
+
+## Pending tests
+
+In the test suites, all but the first test have been skipped.
+
+Once you get a test passing, you can unskip the next one by
+commenting out the relevant `@tag :pending` with a `#` symbol.
+
+For example:
+
+```elixir
+# @tag :pending
+test "shouting" do
+  assert Bob.hey("WATCH OUT!") == "Whoa, chill out!"
+end
+```
+
+Or, you can enable all the tests by commenting out the
+`ExUnit.configure` line in the test suite.
```

Comment:
Maybe should point them to the `test_helper.exs` file?

Reply:
Good idea, the current wording in misleading.